test/fixtures/templates.yml

```diff
@@ -38,6 +38,14 @@ pxekickstart:
     locked: false
     type: ProvisioningTemplate
 
+pxeautoyast:
+    name: suse_pxelinux
+    template: DEFAULT linux~LABEL linux~KERNEL <%%= @kernel %>~APPEND initrd=<%%= @initrd %> ramdisk_size=65536 install=<%%= @mediapath %> autoyast=<%%= foreman_url('provision') %> textmode=1
```

Comment:
I think it's correct as is, as otherwise:
```
  1) Failure:
TFTPOrchestrationTest#test_generate_pxe_template_for_build [test/unit/orchestration/tftp_test.rb:40]:
--- expected
+++ actual
@@ -1 +1 @@
-["default linux", "label linux", "kernel  @kernel %>", "append initrd=<%= @initrd %> ks=<%= foreman_url(\"kickstart\")%> ksdevice=bootif network kssendmac"]
+["default linux", "label linux", "kernel boot/Redhat-6.1-x86_64-vmlinuz", "append initrd=boot/Redhat-6.1-x86_64-initrd.img ks=http://ahost.com:3000/unattended/kickstart ksdevice=bootif network kssendmac"]



  2) Failure:
TFTPOrchestrationTest#test_generate_pxe_template_for_suse_build [test/unit/orchestration/tftp_test.rb:93]:
--- expected
+++ actual
@@ -1 +1 @@
-["DEFAULT linux", "LABEL linux", "KERNEL  @kernel %>", "APPEND initrd=<%= @initrd %> ramdisk_size=65536 install=<%= @mediapath %> autoyast=<%= foreman_url('provision') %> textmode=1"]
+["DEFAULT linux", "LABEL linux", "KERNEL boot/OpenSuse-12.3-x86_64-linux", "APPEND initrd=boot/OpenSuse-12.3-x86_64-initrd ramdisk_size=65536 install=http://download.opensuse.org/distribution/12.3/repo/oss autoyast=http://ahost.com:3000/unattended/provision textmode=1"]
``